packages/ui/lit-ui/src/dx-tag-picker/dx-tag-picker-item.ts

```diff
@@ -0,0 +1,59 @@
+//
+// Copyright 2025 DXOS.org
+//
+
+import { html, LitElement } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+
+import { makeId } from '@dxos/react-hooks';
+
+export class DxTagPickerItemClick extends Event {
+  public readonly itemId: string;
+  public readonly action: 'remove' | 'activate';
+  constructor(props: { itemId: string; action: 'remove' | 'activate' }) {
+    super('dx-tag-picker-item-click');
+    this.itemId = props.itemId;
+    this.action = props.action;
+  }
+}
+
+@customElement('dx-tag-picker-item')
+export class DxTagPickerItem extends LitElement {
+  // TODO(thure): Get Hue type used in theme.
+  @property({ type: String })
+  hue: string = 'neutral';
+
+  @property({ type: String })
+  itemId: string = makeId('dx-tag-picker-item');
+
+  @property({ type: String })
+  label: string = 'never';
```

Comment:
never?

Reply:
@thure Why can't this be string | undefined?